db/migrate/20200302173620_drop_end_user_trigger_procedure.rb

```diff
@@ -0,0 +1,5 @@
+class DropEndUserTriggerProcedure < ActiveRecord::Migration[5.0]
+  def change
+    System::Database::Postgres::TriggerProcedure.new('tp_end_user_plans_tenant_id', nil).drop
+  end
+end
```

Comment:
```suggestion
class DropEndUserTriggerProcedure < ActiveRecord::Migration[5.0]
  def change
    return unless System::Database.postgres?
    System::Database::Postgres::TriggerProcedure.new('tp_end_user_plans_tenant_id', nil).drop
  end
end
```

And then run for all DBs :smile: because it will update the schema number for all schemas :smile:

Reply:
I can run it for all DBs if u want, I don't mind, because I will have to do so for another PR anyway :smile: But add the "return" line please :smile: